static/panes/tree.ts

```diff
@@ -191,9 +191,12 @@ export class Tree {
     }
 
     private paneRenamedExternally() {
-        this.multifileService.forEachFile(
-            (file: MultifileFile) => (file.filename = this.hub.getEditorById(file.editorId)?.getPaneName() ?? ''),
-        );
```

Comment:
When the associated editor is closed, we don't want a tree-initiated rename to empty the `filename`.